### PR TITLE
fix: update embedded score specification and fix reference docs

### PR DESCRIPTION
**Pull request by astromechza**
This PR updates the score specification embedded in these docs and updates the reference docs to fix some missing details and incorrect names (primarily volume read_only).

This was done via a `subptree pull` as documented in the score-spec/schema repo (which is why the makefil and sample were pulled in).

🚀

### CODE REVIEW

File: schemas/score-v1b1.json [hunk 1]
```diff
@@ -72,133 +89,154 @@
...
             "type": "object",
+<<<<<<< HEAD
             "required": [
               "target"
             ],
+=======
+            "required": [
+              "target"
+            ],
+>>>>>>> 95847dd8aa805318cb287c6d25dcddf965f50c26
```
Comment (johanneswuerbach) [inline schemas/score-v1b1.json:228]: Merge relict?

File: schemas/score-v1b1.json [hunk 2]
```diff
@@ -228,28 +254,50 @@
             },
             "oneOf": [
               {
+<<<<<<< HEAD
```
Comment (johanneswuerbach) [inline schemas/score-v1b1.json:257]: More here?

**Review by johanneswuerbach — CHANGES_REQUESTED** [review]
Looks like there are some merge conflicts.

File: schemas/score-v1b1.json (hunk 1 above)
Comment (astromechza) [reply schemas/score-v1b1.json]: fixed up! yup looks like some undetected merge conflict. I've re-copied the schema in from source.

File: schemas/score-v1b1.json (hunk 2 above)
Comment (astromechza) [reply schemas/score-v1b1.json]: fixed up! yup looks like some undetected merge conflict. I've re-copied the schema in from source.

**Comment by astromechza** [comment]
Thanks @johanneswuerbach, fixed up

**Review by johanneswuerbach — APPROVED** [review]

**Pull request merged**